internal/integrations/v4/testhelp/command.go

```diff
@@ -23,10 +23,6 @@ func GoCommand() string {
 	if err != nil {
 		return "cant-find-go:" + err.Error()
 	}
-
```

Comment:
Removing this as it was already replaced in line 22 when calling `gocmd, err := getGoCommand()`